app/views/users/show.html.erb

```diff
@@ -1,55 +1,50 @@
-<div class="container">
-    <div class="row">
-
-        <div class="col-md-12 d-flex ">
-
-            <img src="" height="50">
-
-            <div class="">
-
-                <h3>My name</h3>
-
-                <span>Number of post: 0</span>
-
-            </div>
-
-        </div>
-
+<main class="card">
+  <div class="card-content">
+    <div class="user-image border"></div>
```

Comment:
- [ ] In the the user show page and as per the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png), you are also supposed to display the `user.photo.` Kindly add that to your code